app/workflow/repository.go

```diff
@@ -80,6 +81,28 @@ func (repository *repository) loadMeta(workflow *model.Workflow) (err error) {
 	return
 }
 
+// Save meta information to workflow_meta table from a workflow object.
+func (repository *repository) SaveMeta(workflow *model.Workflow) (err error) {
```

Comment:
I think it would be best if `SaveMeta()` would not be exported, and use the same strategy as in worfklowstep repository, where `saveMeta()` is called by `update()` and `insert()`. We could then also remove `SaveMeta()` from the repository interface.

Reply:
Yeah! I thought about that but since identifier is going to be moved over from meta to main table's column like we discussed (See #71) and we would remove meta table itself in near future, so we would simply remove this method from the interface and the implementation in a follow up PR.

Reply:
As an exercise, I made this change. Earlier I couldn't figure something out somewhere to make it work without exporting. But now its done.